### ISSUE TITLE
BML not working under wine.

### ISSUE THREAD

**Issue by Swung0x48** [Gamepiaynmo/BallanceModLoader#7]
I've attempted to install BML 0.3.31 on macOS Wine(`WS11WineCX64Bit19.0.1-1`).
The game crashes at cold boot. (It works fine without BML, so not Wine's fault, or at least it's not on its own)
Just simply pulling BML out of that Ballance copy made it alive again.

After inspecting `ModLoader.log` , it seems like minhook has failed to hook? 
```
[03/04/2021 08:03:37.473] [ModLoader/INFO]: Initializing Mod Loader version 0.3.30
[03/04/2021 08:03:37.473] [ModLoader/INFO]: Website: https://github.com/Gamepiaynmo/BallanceModLoader
[03/04/2021 08:03:37.473] [ModLoader/INFO]: MinHook Initialized
[03/04/2021 08:03:37.740] [ModLoader/INFO]: Step Hook created
[03/04/2021 08:03:37.740] [ModLoader/ERROR]: Create Object Loader Hook Failed
```
Compiling from source won't help. (which spits out the same error)

**Comment by Swung0x48** [comment]
I found those lines(and a few in similar pattern) in the source code, presumably be the cause of this issue.
```
if (MH_CreateHook(reinterpret_cast<LPVOID>(0x25681C30), &ObjectLoader,
		reinterpret_cast<LPVOID*>(&m_objectLoader)) != MH_OK
		|| MH_EnableHook(reinterpret_cast<LPVOID>(0x25681C30)) != MH_OK) {
		m_logger->Error("Create Object Loader Hook Failed");
		return;
}
```

After digging deeper in OllyDbg, it seems like the addresses has been changed in wine.
![image](https://user-images.githubusercontent.com/16967689/109890122-b1e46600-7cc1-11eb-92f0-2a8e13656d67.png)
Replace the address in original code with the new one, now it hooks like a charm!

```
[03/03/2021 22:49:58.440] [ModLoader/INFO]: Initializing Mod Loader version 0.3.31
[03/03/2021 22:49:58.440] [ModLoader/INFO]: Website: https://github.com/Gamepiaynmo/BallanceModLoader
[03/03/2021 22:49:58.440] [ModLoader/INFO]: MinHook Initialized
[03/03/2021 22:49:58.765] [ModLoader/INFO]: Step Hook created
[03/03/2021 22:49:58.833] [ModLoader/INFO]: Object Loader Hook created
[03/03/2021 22:49:58.833] [ModLoader/ERROR]: Create Input Manager Hooks Failed
```
(Too lazy to find out all of the replacements, but still a solid proof of concept)
Though I dunno if there's a cleaner way to dynamically get that address... But it seems the root cause of the problem.

**Comment by Gamepiaynmo** [comment]
Fixed in 0.3.33.
